src/components/Sequencer/index.tsx

```diff
@@ -1,8 +1,27 @@
 import * as React from "react";
 import {SectionContainer} from "../sharedStyles";
+import { SequencerButton } from "../SequencerButton";
+import { SequencerStyles } from "./styles";
 
 export interface ISequencerProps {}
 
 export default function Sequencer(props: ISequencerProps) {
-  return <SectionContainer>Sequencer</SectionContainer>;
+  return (
+    <SequencerStyles>
+      <SectionContainer>
+        Sequencer
+        <SequencerButton></SequencerButton>
```

Comment:
We can make the sequencer button smart so it doesn't need a closing element.

```
<SequencerButton />
```

To do this, in the sequencer component we would check for children:

```
const SequencerComponent = (children) => {
  if (children) {
    return <StyledButton>{children}<StyledButton>
  }

  // No children
  return <StyledButton />
}
```

Reply:
didn't quite get this to work...i think i tried to render it conditionally on the `SequencerButton` component instead of the sequencer. wasn't totally comfortable refactoring the sequencer component as there's some typescripty stuff i didn't fully understand